low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -170,22 +173,40 @@ def _raid_health_monitor(self):
                 # Check RAID device array state is 'idle' or not
                 result = self._check_raid_state(device)
                 if result == "failed":
```

Comment:
Having just this check will still send alert again across system/SSPL resets. That can be handled at lower priority, I think.